src/server/api/routers/message.ts

```
@@ -40,6 +42,9 @@ export const messageRouter = createTRPCRouter({
             }
           },
         });
+
+        void captureEvent(ctx.session.user.id, 'message_sent', { thread_id: input.threadId, platform: input.platform ?? 'web' });
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'platform' should be before 'thread_id'.

(sort-keys-fix/sort-keys-fix)